apps/src/netsim/NetSimBitLogPanel.html.ejs

```diff
@@ -1,5 +1,5 @@
 <%
-  var i18n = require('./locale');
+  var i18n = require('@cdo/netsim/locale');
```

Comment:
Is this related to lodash? Do we have guidance on when to use @cdo vs using a relative path?